target/linux/mediatek/patches-4.14/0229-fix-memory-size-for-bpi-r2.patch

```diff
@@ -0,0 +1,13 @@
+Index: linux-4.9.44/arch/arm/boot/dts/mt7623n-bananapi-bpi-r2.dts
+===================================================================
+--- linux-4.9.44.orig/arch/arm/boot/dts/mt7623n-bananapi-bpi-r2.dts	2017-09-02 08:21:11.998199513 +0800
++++ linux-4.9.44/arch/arm/boot/dts/mt7623n-bananapi-bpi-r2.dts	2017-09-02 08:33:09.452252584 +0800
+@@ -18,7 +18,7 @@
+ 	};
+ */
+ 	memory {
+-		reg = <0 0x80000000 0 0x20000000>;
++		reg = <0 0x80000000 0 0x80000000>;
```

Comment:
patch the dts file directly

Reply:
What do you mean?

Reply:
@blogic Ping?

Reply:
@abbradar: `mt7623n-bananapi-bpi-r2.dts` is upstream DTS file, if there is something wrong in it, please send upstream patch for that

Reply:
at least for 4.19+ this is already fixed in upstream

https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/arm/boot/dts/mt7623n-bananapi-bpi-r2.dts?h=v4.19#n111

in 4.14 it is set to different value than these 2

https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable-rc.git/tree/arch/arm/boot/dts/mt7623n-bananapi-bpi-r2.dts?h=linux-4.14.y#n104